common/src/main/java/com/wynntils/models/abilities/bossbars/AwakenedBar.java

```diff
@@ -10,7 +10,7 @@
 import java.util.regex.Pattern;
 
 public final class AwakenedBar extends TrackedBar {
-    private static final Pattern AWAKENED_PATTERN = Pattern.compile("§fAwakening §7\\[§f(\\d+)/(\\d+)§7]");
+    private static final Pattern AWAKENED_PATTERN = Pattern.compile("§fAwakened §7\\[§f(\\d+)/(\\d+)§7]");
```

Comment:
Just checking, did Wynn change their string or was this always wrong?

Reply:
There's reports of it not working as far back as July last year so I presume it was changed before then, I've only just leveled a shaman myself so can't confirm when it was changed

Reply:
This used to work baaack then.